packages/ng-schematics/src/component/index_spec.ts

```diff
@@ -23,8 +24,6 @@ describe("component",  () => {
 		const tree = runner.runSchematicAsync("component",
 			{ name: "my-combo", template: "combo", templateInst: mockInst, skipRoute: false }, Tree.empty());
 		tree.subscribe(state => {
-			expect(mockInst.generateConfig).toHaveBeenCalledWith("my-combo", {});
-			expect(mockInst.registerInProject).toHaveBeenCalledWith("", "my-combo", { skipRoute: false, modulePath: undefined });
```

Comment:
Erm, shouldn't `registerInProject` still be called?

Reply:
Yeah, you're correct, this was actually failing for a reason - I've reverted the change here are adjusted the schematics code